app/uk/gov/hmrc/cdsreimbursementclaimfrontend/views/claims/enter_duplicate_mrn_or_entry_number.scala.html

```diff
@@ -51,8 +51,8 @@
 
             @inputText(
                 form = form,
-                id = key,
-                name = key,
+                id = "enter-duplicate-mrn-or-entry-number",
+                name = "enter-duplicate-mrn-or-entry-number",
```

Comment:
same comment as above - do not hardcode the message key at this point

Reply:
See my comment above